lib/medtronic600/medtronic600Simulator.js

```diff
@@ -510,13 +382,122 @@ class Medtronic600Simulator {
     });
   }
 
+  applySuspendedBasals() {
```

Comment:
I didn't take offence :) I think we're saying the same thing. It's definitely a good idea to give Eric the use cases so that he can generate the data for his own tests.